server/service/handler.go

```diff
@@ -727,6 +722,8 @@ func attachNewStyleFleetAPIRoutes(r *mux.Router, svc fleet.Service, opts []kitht
 	e.GET("/api/v1/fleet/software", listSoftwareEndpoint, listSoftwareRequest{})
 
 	e.POST("/api/v1/fleet/hosts/delete", deleteHostsEndpoint, deleteHostsRequest{})
+	e.GET("/api/v1/fleet/hosts/count", countHostsEndpoint, countHostsRequest{})
+	e.GET("/api/v1/fleet/hosts/{id}", getHostEndpoint, getHostRequest{})
```

Comment:
I also took a chance and migrated this endpoint from the old form because it was taking precedence.